Dockerfile

```diff
@@ -24,20 +25,42 @@ ENV REACT_APP_VERSION=$REACT_APP_VERSION
 ARG REACT_APP_COMMIT
 ENV REACT_APP_COMMIT=$REACT_APP_COMMIT
 
+# Build production application files
 RUN yarn build
```

Comment:
Is there any compelling reason to use `yarn` instead of `npm`? As you know, elsewhere we have standardized on `npm`

Reply:
Thats what the boilerplate authored by Ivan was using. I think he favored yarn for in house projects which carried over here.
I'll have to see about switching it over to the mojaloop standard another time.